pkg/apis/v1alpha1/wait.go

```diff
@@ -0,0 +1,65 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// OutputFormatType defines the enum for valid output formats.
+type OutputFormatType string
```

Comment:
We already have something defined for that (see in other collectors), we only support yaml and json for now.